_includes/home/homepage-startersexamples-band.html

```diff
@@ -3,8 +3,10 @@
   <div class="highlights-alternating-inline-block">
     <div class="grid-wrapper">
       <div class="width-8-12 width-12-12-m block-content mobile-flip">
-        <h2>STARTERS & EXAMPLES</h2>
-        <p>The Nodeshift project contains a wide range of starter applications showcasing the most basic cloud native concepts, like REST microservices and Health Checks, to more advanced concepts like Cloud Native Security.</p>
+        <h2>Reference Architecture & Examples</h2>
+        <p>The goal of the Node.js Reference Architecture is to present the Nodeshift teams opinion on what components to use when building Node.js applications and guidance on how to be succesful in a production environment.</p>
```

Comment:
```suggestion
        <p>The goal of the Node.js Reference Architecture is to present the Red hat and IBM teams opinion on what components to use when building Node.js applications and guidance on how to be succesful in a production environment.</p>
```

Reply:
I think this should be a bit broader than the "Nodeshift" team as it is meant to reflect the broader opinion.

Reply:
that sounds reasonable